1회차/이유주/2_퍼펙트셔플.py

```diff
@@ -1,3 +1,28 @@
 import sys
 
 sys.stdin = open("_퍼펙트셔플.txt")
+
+T = int(input()) #테스트케이스 수
+
+
+
+for i in range(1, T+1):
+    n = int(input()) #카드 수
+    divide = [0] * n
+    card = list(map(str, input().split()))
+   
+
+    #짝수일때
+    if n % 2 == 0:
+        for x in range(n//2):
+            divide[x*2] = card[x]
+            divide[(x*2)+1] = card[(n//2)+x]
+        
+    # #홀수일때
+    if n % 2 == 1:
+        for x in range(n//2+1):
+            divide[x*2] = card[x]
+        for x in range(n//2):
+            divide[x*2+1] = card[(n//2)+1+x]
+    print(f'#{i} {divide}')
+
```

Comment:
경우를 나눠서 정말 잘 해결하셨네요! 👍 
문제가 요구하는 출력 양식에 맞춰서 출력하도록 변경만 해주면 될 것 같습니다!
```suggestion
    print(f'#{i}', *divide)

```